romancal/refpix/tests/test_refpix.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+from astropy import units as u
+
+from romancal.refpix.data import StandardView
+from romancal.refpix.refpix import Control, run_steps
+
+from . import reference_utils
+
+
+# GitHub actions don't have enough memory to run this test
+@pytest.mark.bigdata
```

Comment:
See https://github.com/spacetelescope/romancal/pull/704#issuecomment-1603123731. This correction is very much tied to the physical configuration of the detectors and so it is not simple to use smaller arrays.

Reply:
Playing devil's advocate briefly, I think the reference pixel correction algorithm is probably pretty well behaved in the number of rows, so you might be able to make a fake exposure shape that instead of being 4096 rows long were only ~12 rows long.  I think you only need to be >8, otherwise things stop making sense due to padding?

Reply:
@schlafly that will actually work. Though it will be a pain because the reference code I am testing against makes a hard assumption on the number of rows...

Reply:
@schlafly your suggestion will work, see 310cc414628d3026e9d5c399216e8135dbd2a07c. However, it required several modifications to the Goddard code we are testing against. If you are fine with those changes then we can keep them.

Reply:
For my two cents those changes look pretty sane and look valuable to me for unit test purposes.